Portscout/Config.pm

```diff
@@ -104,6 +104,7 @@ my (@paths, %settings_types, $bool_opts);
 	mail_enable     	=> 1,
 	mail_from       	=> APPNAME,
 	mail_subject    	=> '',
+	mail_altsubject    	=> '',
```

Comment:
```suggestion
	mail_subject    	=> 'FreeBSD ports you maintain which are out of date',
	mail_altsubject    	=> 'Unmaintained FreeBSD ports which are out of date',
```

(I would suggest to set both, otherwise the next contributor will be confused why one and not the other)

Reply:
Unfortunately it seems like `portscout` completely ignores the default values in `Config.pm` and looks for them in `portscout.conf` anyway, seems like a bug in the config file parser.

Reply:
> > Unfortunately it seems like `portscout` completely ignores the default values in `Config.pm` and looks for them in `portscout.conf` anyway, seems like a bug in the config file parser.
> 
> Hm, I just tried locally and I could use defaults no problem (tested by commenting out `mail subject` in portscout.conf and modifying `mail_subject` default in Config.pm). I then output the content of %settings in portscout.pl and it work as expected (showing what I set in portscout.conf if it is configured and the default if nothing was set).
> 
> I would suspect that you're either loading Config.pm from the wrong location (/usr/local/lib/perl5/site_perl/Portscout/Config.pm instead of your working copy) or that you're loading the wrong portscout.conf in testing.

Whoops, yes, you're absolutely right, I was loading the wrong module. My bad.

Well, then problem solved I guess. Seems to me like a nice solution.